packages/framework-types/src/config.ts

```diff
@@ -33,6 +33,7 @@ export class BoosterConfig {
   public rockets?: Array<RocketDescriptor>
   public appName = 'new-booster-app'
   public assets?: Array<string>
+  public defaultResponseHeaders?: Record<string, string>
```

Comment:
Unless you're initializing this somewhere else, I think that we should provide a default value here:

```suggestion
  public defaultResponseHeaders: Record<string, string> = {}
```